packages/react-log-viewer/package.json

```
@@ -1,5 +1,5 @@
 {
-  "name": "@patternfly/react-log-viewer",
+  "name": "@patternfly-react/react-log-viewer",
```

Should this be `@patternfly-five`?
```suggestion
  "name": "@patternfly-five/react-log-viewer",
```